tool/site/client-react/src/pages/returnPage.tsx

```diff
@@ -0,0 +1,217 @@
+import * as React from "react"
+import {useContext, useState} from "react"
+import {Label, TickIcon, CrossIcon, Form, Button, Fieldset, SummaryList} from "nhsuk-react-components"
+import {AppContext} from "../index"
+import ButtonList from "../components/buttonList"
+import LongRunningTask from "../components/longRunningTask"
+import * as fhir from "fhir/r4"
+import MessageExpanders from "../components/messageExpanders"
+import ReloadButton from "../components/reloadButton"
+import {axiosInstance} from "../requests/axiosInstance"
+import {getResponseDataIfValid} from "../requests/getValidResponse"
+import {ApiResult, isApiResult} from "../requests/apiResult"
+import {Formik} from "formik"
+import BackButton from "../components/backButton"
+import * as uuid from "uuid"
+import {formatCurrentDateTimeIsoFormat} from "../formatters/dates"
+import PharmacyRadios from "../components/pharmacies"
+import RadioField from "../components/radioField"
+
+interface ReturnPageProps {
+  prescriptionId?: string
+}
+
+interface ReturnFormValues {
```

Comment:
would you mind moving the returnForm out of the return page? just helps keep files small

Reply:
moved